### ISSUE TITLE
Check GitHub Actions GHC versions match tested-with

### ISSUE THREAD

**Issue by tomjaguarpaw** [ndmitchell/neil#51]
It would be good to get the equivalent of this check for GitHub actions.

(This issue is mostly for informational purposes -- I don't find this particularly critical)

https://github.com/ndmitchell/neil/blob/f49dd93a2ec9f4eb2a2b2a95d1918959c46c6cbf/src/Cabal.hs#L91-L93

**Comment by ndmitchell** [comment]
Yep, I need to tidy up the whole process. My hope is that https://github.com/actions/runner/issues/646 stabilises first. I view this as slightly useful, but not that important for a good few months (its the kind of thing that drifts over time, but in the short term, isn't a bit deal).
